app/routes/index.tsx

```diff
@@ -134,28 +137,94 @@ export default function Index() {
                 alt="Open in Gitpod"
                 width="200"
                 height="56"
-                src="/open-in-gitpod.svg"
+                src="https://gitpod.io/button/open-in-gitpod.svg"
```

Comment:
I’d prefer to leave this self hosted, it saves a few hundred ms on the image load for cache misses
```suggestion
                src="/open-in-gitpod.svg"
```

Reply:
Ok, i will upload that file and update this line in next commit.

Reply:
resolved in [`d0d5120`](https://github.com/jacobparis/gitpodify/pull/2/commits/d0d512008b4b7045faef18c1661b75e66c2fcf88)